department-of-veterans-affairs/va.gov-cms
# VACMS-2533 fix paragraph test and reformat all tests

## Description
See #2533 
This pr adds a patch to the TableEqualityAssertion class required by spec tool. At present, when duplicate entries are detected in a .feature file, an ambiguous / unhelpful error is thrown. This pr outputs a message indicating a duplicate has been detected, and the duplicate line.

This pr also reformats dst .feature files to match the formatting in the spec tool google sheet.

Depends on https://github.com/department-of-veterans-affairs/va.gov-cms/pull/2567

## Testing done
Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/89533724-d6c33a00-d7c1-11ea-990d-a174af552c39.png)

## QA steps
Add a duplicate line to any dst feature file and run the test - visually verify that the duplicate line message and line appear in the error message.
